examples/p4est_2d_dgsem/elixir_advection_basic.jl

```diff
@@ -31,7 +31,7 @@ semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition_convergen
 # ODE solvers, callbacks etc.
 
 # Create ODE problem with time span from 0.0 to 1.0
-ode = semidiscretize(semi, (0.0, 1.0))
+ode = semidiscretize(semi, (0.0, 1.0); real_type = nothing, storage_type = nothing)
```

Comment:
Could we maybe use another elixir like the `_extended` version to avoid polluting the simplest elixirs?